src/Middleware/WebSockets/src/ExtendedWebSocketAcceptContext.cs

```diff
@@ -6,13 +6,23 @@
 
 namespace Microsoft.AspNetCore.WebSockets
 {
+    /// <summary>
+    /// Extends the <see cref="WebSocketAcceptContext"/> class with additional properties.
+    /// </summary>
     public class ExtendedWebSocketAcceptContext : WebSocketAcceptContext
     {
+        /// <inheritdoc />
         public override string SubProtocol { get; set; }
 
+        /// <summary>
+        /// This property is obsolete and has no effect.
+        /// </summary>
         [Obsolete("Setting this property has no effect. It will be removed in a future version.")]
         public int? ReceiveBufferSize { get; set; }
 
+        /// <summary>
+        /// The interval to send pong frames. This is a heart-beat that keeps the connection alive.
```

Comment:
> pong frames

That's not a typo, is it?

Reply:
Nope, the websockets impl specifically sends pong frames instead of ping frames.

Reply:
Wait, where? I've only seen that in WinHttp.

Reply:
Oh, https://github.com/dotnet/runtime/blob/b5705587347d29d79cec830dc22b389e1ad9a9e0/src/libraries/Common/src/System/Net/WebSockets/ManagedWebSocket.cs#L515-L517